Tests/FuzzilliTests/LifterTest.swift

```diff
@@ -535,6 +535,41 @@ class LifterTests: XCTestCase {
 
         XCTAssertEqual(lifted_program,expected_program)
     }
+
+    func testAssignmentOperationLifting(){
```

Comment:
Nice, thanks for adding tests! Can you please also extend [this](https://github.com/googleprojectzero/fuzzilli/blob/7dbaa342b2fe00da8f33c8d96c7b5eb460e24fdc/Tests/FuzzilliTests/InterpreterTest.swift#L606) test? (Just add another loop and pretty much copy everything else from the BinaryOperation case)